website/content/en/preview/getting-started/scripts/step15-delete-node.sh

```diff
@@ -0,0 +1 @@
+kubectl delete node $NODE_NAME
```

Comment:
I'd just move this to a comment in the provisioning script or the README since someone would have to go into this file or set $NODE_NAME themselves for one command.
```suggestion
# Use this to delete one node
kubectl delete node $NODE_NAME
# Use this to delete all nodes created by a provisioner
kubectl delete node -l karpenter.sh/provisioner-name=$PROVISIONER_NAME
```

Reply:
The readme does mention that.